marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/WriteStepRunner.java

```diff
@@ -539,6 +539,15 @@ private RunStepResponse runIngester(RunStepResponse runStepResponse, Collection<
             while(itr.hasNext()) {
                 try {
                     File file = new File((String) itr.next());
+                    if("csv".equalsIgnoreCase(inputFileType)) {
+                        //Add "fileName" to options so that it gets added to header in enode code
+                        options.put("fileName", file.getAbsolutePath());
```

Comment:
I would call this "file" or "filePath" or "pathname" instead of "fileName", as fileName/filename typically imply just the name of the file and not the full path. 

This would be consistent with the only other place where "filename" is used in DHF - in the tasks for importing/exporting jobs. In those tasks, "filename" is just for the name of a file, not including the path.